cas-client-core/src/main/java/org/jasig/cas/client/util/ReflectUtils.java

```diff
@@ -148,4 +149,35 @@ public static void setProperty(final String propertyName, final Object value, fi
             throw new RuntimeException("Error setting property " + propertyName, e);
         }
     }
+
+    /**
+     * Gets the value of the given declared field on the target object or any of its superclasses.
+     *
+     * @param fieldName Name of field to get.
+     * @param target Target object that possesses field.
+     *
+     * @return Field value.
+     */
+    public static Object getField(final String fieldName, final Object target) {
+        Class<?> clazz = target.getClass();
+        Field field = null;
+        do {
+            try {
+                field = clazz.getDeclaredField(fieldName);
+            } catch (NoSuchFieldException e) {
+                clazz = clazz.getSuperclass();
+            }
+        } while (field == null && clazz != null);
+        if (field == null) {
+            throw new IllegalArgumentException(fieldName + " does not exist on " + target);
+        }
+        try {
+            if (!field.isAccessible()) {
+                field.setAccessible(true);
```

Comment:
No. My reading of the Javadocs suggest it's just for the reference, https://docs.oracle.com/javase/7/docs/api/java/lang/reflect/AccessibleObject.html#setAccessible(java.lang.reflect.AccessibleObject[],%20boolean); moreover, I've never seen it done in practice.